### ISSUE TITLE
[NW2 issue] : Resizing two binded windows doesn't work anymore

### ISSUE THREAD

**Issue by DimPaDev** [nwjs/nw.js#7369]
NWJS Version : 0.44.0 
Operating System : Linux (Fedora 31, both kde and gnome), Windows 10

### Expected behavior

When you bind two windows both in size and position you should be able also to resize one window and the other should correctly react to the changes defined in the 'resize' event just like in nw1 mode (--disable-features=nw2).

![nw1resize](https://user-images.githubusercontent.com/42065117/74186489-03e7c300-4c4b-11ea-94b3-c41e00ea79fc.gif)

### Actual behavior

In Nw2 mode if you bind two windows in size using the 'resize' event and its callback parameters results in a weird flickering issue which prevents the application to work normally.

![nw2resize](https://user-images.githubusercontent.com/42065117/74186687-575a1100-4c4b-11ea-91ca-843c9013008b.gif)

I also found that this code will work even in nw2 mode : 

```
             secondWin.on('resize', (w, h) => {
              console.log('ok main win resized');
              mainWin.resizeTo(w, h);
            });
```

However, if you try to add something to the resizeTo call the bug will appear: 

```
            secondWin.on('resize', (w, h) => {
              console.log('ok main win resized');
              // with this line the bug appears in nw2 mode
              mainWin.resizeTo(w + 50, h + 50);
            });
```

### How to reproduce

DEMO:  [nw2-resize-two-windows-bug.zip](https://github.com/nwjs/nw.js/files/4182879/nw2-resize-two-windows-bug.zip)

**Comment by DimPaDev** [comment]
As it is a blocking issue I will reference this to : #7230

**Comment by rogerwang** [comment]
This was also fixed with #7368

**Comment by DimPaDev** [comment]
I can confirm this is fixed, thanks!
